charts/all-in-one/templates/rudolf-service.yaml

```diff
@@ -0,0 +1,67 @@
+{{ if .Values.rudolfService.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    eks.amazonaws.com/role-arn: {{ .Values.rudolfService.serviceAccount.roleArn }}
+  labels:
+    app.kubernetes.io/instance: {{ $.Chart.Name }}
+  name: {{ $.Chart.Name }}-9c-rudolf-iam-role
+  namespace: {{ $.Chart.Name }}
```

Comment:
`$.Chart.Name` is always fixed to `9c-network`. so if you want to make this fields follow network name, you'd better use `$.Release.Name`

Reply:
Thanks. I'll update the all related parts. 🙏🏻

Reply:
I applied it in [c88b69e](https://github.com/planetarium/9c-infra/pull/787/commits/c88b69ec31fd2d16f1c226b846cbfb8426688b5c) commit.